src/clappr-dash-shaka-playback.js

```diff
@@ -449,6 +457,7 @@ class DashShakaPlayback extends HTML5Video {
       bandwidth: activeVideo.bandwidth,
       width: activeVideo.width,
       height: activeVideo.height,
+      language: activeVideo.language,
```

Comment:
Hi @iSchluff ! Exists one specific need for passing the `language` here?

Reply:
This is necessary for a language switcher plugin to actually know the current audio language.
E.g. https://github.com/voc/clappr-audio-track-selector-plugin/blob/052068eb396a4e4cbe98482e8719aa2f24281fb2/src/main.js#L144

Another possibility would be to add an additional method on the playback to request the current language.

Reply:
Got it. Maybe we can create one event to trigger when the audio is changed (including when the playback sets the initial audio), but for this moment I think this is ok.